llama-index-core/llama_index/core/chat_engine/context.py

```diff
@@ -69,6 +69,7 @@ def from_defaults(
         prefix_messages: Optional[List[ChatMessage]] = None,
         node_postprocessors: Optional[List[BaseNodePostprocessor]] = None,
         context_template: Optional[str] = None,
+        llm: Optional[LLM] = None,
```

Comment:
below, we should also add `llm = llm or llm_from_settings_or_context(...)`

Reply:
Good catch, thanks!